website/src/components/print-lesson.css.ts

```diff
@@ -0,0 +1,144 @@
+import { globalStyle, style } from "@vanilla-extract/css"
+import { important, margin, padding } from "polished"
+import { colors, mediaQueries, thickness } from "src/style/constants"
+import { hsize } from "src/style/constants"
+import { media } from "src/style/utils"
+
+// Parent class defining a Wordpress page.
+// All classes defined in this file begin with this generated className to control the scope of these styles.
+export const wp = style({})
+
+// Displays element on print.
+const showOnPrint = `${wp} .show-on-print`
+globalStyle(showOnPrint, {
+  ...media(mediaQueries.print, {
+    ...important({
+      display: "block",
+    }),
+  }),
+})
+
+// Formats colors of links.
+const link = `${wp} a`
+globalStyle(link, {
+  ...media(mediaQueries.print, {
+    ...important({
+      color: colors.link,
+    }),
+  }),
+})
+
+//// PROSE & SECTIONS
+// Prose is an element that needs to take up the full-width of a page.
+const prose = `${wp} .prose`
+// Section is an element that should not be broken up between pages.
+const section = `${wp} .section`
+
+// This removes line breaks to create full-width pages.
+globalStyle(`${prose} br`, {
+  ...media(mediaQueries.print, {
```

Comment:
Does the `globalStyle` function stop you from doing `globalStyle(..., media(mediaQueries.print, ...))` directly?

Reply:
yeah, I have to do `...media()` anytime I want to use that function. since the `...media()` makes the code look a bit cluttered, I can create a function within this file that does the `...media()` call which the globalStyles will call instead, which may look cleaner.

Reply:
oh actually I can do `globalStyle(..., media(mediaQueries.print, ...))` just fine, I just had to get rid of the `{}` between `...media()`. I can still go ahead and just create a function since there's a lot of duplicate code involving `mediaQueries.print`